docs/analyze/save-and-retrieve.ipynb

```diff
@@ -0,0 +1,352 @@
+    "\n",
+    "To demonstrate, the following cell sets up our Qiskit Runtime service and selects a backend."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 5,
+   "id": "5f5aaa14-910f-401c-b109-73ffc2450f3c",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from qiskit_ibm_runtime import QiskitRuntimeService, SamplerV2 as Sampler\n",
+    "service = QiskitRuntimeService()\n",
+    "backend = service.least_busy(operational=True, min_num_qubits=2)\n",
+    "sampler = Sampler(backend=backend)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "02623345-c252-4963-be89-a074d4a10f54",
+   "metadata": {},
+   "source": [
+    "The following cell submits the job and prints its ID."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 6,
+   "id": "721a5198-a650-4db9-b041-b3a569c6dc20",
+   "metadata": {
+    "tags": [
+     "ignore-warnings"
+    ]
+   },
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "co24umeh1p25p1m5nu7g\n"
+     ]
+    },
+    {
+     "name": "stderr",
+     "output_type": "stream",
+     "text": [
+      "/Users/qiskit/.venv/python3.11/site-packages/qiskit_ibm_runtime/qiskit_runtime_service.py:879: UserWarning: Cloud simulators will be deprecated on 15 May 2024. Use the new local testing mode in qiskit-ibm-runtime version 0.22.0 or later to meet your debugging needs.\n",
+      "  warnings.warn(warning_message)\n"
+     ]
+    }
+   ],
+   "source": [
+    "from qiskit import transpile\n",
+    "job = sampler.run(transpile(qc, backend), shots=10)\n",
+    "my_id = job.job_id()\n",
+    "print(my_id)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "fb271ce3-3e86-476c-ae4e-66df36ba7406",
+   "metadata": {},
+   "source": [
+    "Now we have the job's ID, we can retrieve it in a different session or at a later date using the `service.job` method."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 7,
+   "id": "da02c7ca-803a-4426-8dd0-bbe062c0e9e1",
+   "metadata": {},
+   "outputs": [
+    {
+     "data": {
+      "text/plain": [
+       "PrimitiveResult([PubResult(data=DataBin<>(meas=BitArray(<shape=(), num_shots=10, num_bits=2>)), metadata={'circuit_metadata': {}})], metadata={'version': 2})"
+      ]
+     },
+     "execution_count": 7,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "retrieved_job = service.job(my_id)\n",
+    "retrieved_job.result()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "9aae5f5a-a543-493c-9bc5-5682ba846ab5",
+   "metadata": {},
+   "source": [
+    "### Programatically find jobs\n",
+    "\n",
+    "If you don't have a job ID and want to find it programatically rather than visiting the [Jobs dashboard](https://quantum.ibm.com/jobs), you can use the [`QiskitRuntimeService.jobs`](/api/qiskit-ibm-runtime/qiskit_ibm_runtime.QiskitRuntimeService#jobs) method.\n",
```

Comment:
```suggestion
    "### Programmatically find jobs\n",
    "\n",
    "If you don't have a job ID and want to find it programmatically rather than visiting the [Jobs dashboard](https://quantum.ibm.com/jobs), you can use the [`QiskitRuntimeService.jobs`](/api/qiskit-ibm-runtime/qiskit_ibm_runtime.QiskitRuntimeService#jobs) method.\n",
```